### DISCUSSION TITLE
New release of `jupyter-cache`

### DISCUSSION THREAD

**Discussion by czgdp1807** [executablebooks/meta#1233] [General]
Hi folks. I am starting this discussion for requesting a new release of `jupyter-cache` under `executablebooks` organisation. Actually https://github.com/executablebooks/jupyter-cache/pull/117 got merged. I created it for https://github.com/scipy/scipy/pull/21433. https://github.com/scipy/scipy/pull/21433 can only be merged once a new release of `jupyter-cache` is available on conda and PyPI. Is there any tentative date for a new release? 

Apologies if someone already requested for a new release.

Waiting for maintainers' response.

@agoose77 @chrisjsewell @choldgraf

**Comment by czgdp1807** [comment]
@agoose77 @chrisjsewell @choldgraf Waiting for your response on this. Thank you.

**Reply by agoose77** [reply]
Thanks for the heads up.

I'll look to make a release on Monday.

**Reply by czgdp1807** [reply]
@agoose77 A gentle reminder for this.

**Reply by czgdp1807** [reply]
Hi. Is the release process on for this? @agoose77

**Reply by choldgraf** [reply]
We will get to it when there is time, please be patient. Right now our priority is releasing an alpha of jupyter book 2.0.

**Reply by agoose77** [reply]
@choldgraf thanks for advocating for our team capacity Chris! In this particular instance, I will caveat that I did suggest I'd get to it on Monday, so I can understand @czgdp1807's follow up. 

In general, @czgdp1807, @choldgraf is correct that we are focusing our efforts on Jupyter Book 2.

**Reply by agoose77** [reply]
@czgdp1807 v1.0.1 should be on PyPI soon :crossed_fingers:

**Reply by czgdp1807** [reply]
No worries. Take your time for a new release. ❤️.

**Comment by czgdp1807** [comment]
Closing this discussion since the new release is available and the linked SciPy PR is merged.
